go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -159,6 +161,15 @@ func ensureClusterRaftMembership(db string) {
 	r, _ := regexp.Compile(`([a-z0-9]{4}) at ((ssl|tcp):\[?[a-z0-9.:]+\]?)`)
 	members := r.FindAllStringSubmatch(out, -1)
 	kickedMembersCount := 0
+	masterNameLabel := map[string]string{"name": "ovnkube-master"}
```

Comment:
the dbs could be on nodes that are not ovnkube-master right? So do we need to search a different way?

Reply:
yeah, good point. so we may have to add some other label on the ovnkube-db pods in upstream and ovnkube-master pods in Openshift to track the pods that run the raft dbs.